src/persistence.js

```diff
@@ -1058,6 +1063,9 @@ export default function _Persistence(mpInstance) {
     };
 
     this.getPersistence = function() {
+        if (!this.isEnabled()) {
```

Comment:
See comment about about not using the param on the constructor.  I am open to feedback about this decision.  I'd rather just have it consistent everywhere though so that it's easier to grok, even if it's more verbose.